notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<ol><li>Let's give an explanatory title with an H1 header. E.g. "How to use inference endpoints to embed documents". This way, it's easier for people to know what to expect. Then, in the introduction, you can proceed to explain the goal and the approach.</li><li>There are some typos in the text. "Im using" -&gt; "I'm using". </li><li>The sentence " Since it has such long entries, I will use the new <a href="https://huggingface.co/jinaai/jina-embeddings-v2-base-en" target="_blank">jinaai/jina-embeddings-v2-base-en</a> since it has an 8k context length. Since Im GPU-poor I will deploy this using <a href="https://huggingface.co/inference-endpoints" target="_blank">Inference Endpoint</a> to save money and time." is a bit repetitive ("since"). Here's a suggestion: " Because it has long entries, I will use the new <a href="https://huggingface.co/jinaai/jina-embeddings-v2-base-en" target="_blank">jinaai/jina-embeddings-v2-base-en</a> since it has an 8k context length. I will deploy this using <a href="https://huggingface.co/inference-endpoints" target="_blank">Inference Endpoint</a> to save money and time."</li><li>"To follow this you will need..." -&gt; "To follow this tutorial, you will need ..."</li><li>"If you haven't add one here in <a href="https://huggingface.co/docs/hub/billing#billing" target="_blank">billing</a>." -&gt; "If you haven't, you can add one here in <a href="https://huggingface.co/docs/hub/billing#billing" target="_blank">billing</a>." (this reads a little easier)</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Good ideas!